src/components/plotNFT/PlotExternalNFTCard.tsx

```diff
@@ -282,7 +282,7 @@ export default function PlotExternalNFTCard(props: Props) {
 
           <Flex flexDirection="column" gap={2} flexGrow={1}>
             <Flex flexDirection="column" flexGrow={1}>
-              <CardKeyValue rows={rows} hideDivider />
+              <CardKeyValue rows={rows} $hideDivider />
```

Comment:
$hideDivider does not exists

Reply:
let me find the documentation i was referring too